daprdocs/content/en/reference/components-reference/supported-state-stores/setup-oracledatabase.md

```diff
@@ -0,0 +1,165 @@
+---
+type: docs
+title: "Oracle Database"
+linkTitle: "Oracle Database"
+description: Detailed information on the Oracle Database state store component
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-oracledatabase/"
+---
+
+## Create a Dapr component
+
+Create a component properties yaml file, for example called `oracle.yaml` (but it could be named anything ), paste the following and replace the `<CONNECTION STRING>` value with your connection string. The connection string is a standard Oracle Database connection string, composed as: `"oracle://user/password@host:port/servicename"` for example `"oracle://demo:demo@localhost:1521/xe"`. In case you connect to the database using an Oracle Wallet, you should specify a value for the `oracleWalletLocation` property, for example: `"/home/app/state/Wallet_daprDB/"`; this should refer to the local file system directory that contains the file `cwallet.sso` that is extracted from the Oracle Wallet archive file.
```

Comment:
```suggestion
Create a component properties yaml file, for example called `oracle.yaml` (but it could be named anything ), paste the following and replace the `<CONNECTION STRING>` value with your connection string. The connection string is a standard Oracle Database connection string, composed as: `"oracle://user/password@host:port/servicename"` for example `"oracle://demo:demo@localhost:1521/xe"`. 

In case you connect to the database using an Oracle Wallet, you should specify a value for the `oracleWalletLocation` property, for example: `"/home/app/state/Wallet_daprDB/"`; this should refer to the local file system directory that contains the file `cwallet.sso` that is extracted from the Oracle Wallet archive file.
```

Reply:
will fix